src/components/molecules/NoteDetailFolderNavigator.tsx

```diff
@@ -54,12 +56,17 @@ interface FolderData {
   pathname: string
 }
 
-const NoteDetailFolderNavigator = ({
-  storageId,
-  storageName,
-  noteId,
-  noteFolderPathname,
-}: NoteDetailFolderNavigatorProps) => {
+interface FolderNavItemProps {
+  path: string
```

Comment:
storageId, storageName, noteId, noteFolderPathname are already resolved from parent component. We don't need to parse again. It will make our app slower.